db/changes.go

```diff
@@ -41,6 +41,7 @@ type ChangeEntry struct {
 	ID         string      `json:"id"`
 	Deleted    bool        `json:"deleted,omitempty"`
 	Removed    base.Set    `json:"removed,omitempty"`
+	pseudoDoc  bool        `json:"[pseudo,omitempty` // Used to indicate _user docs e.t.c
```

Comment:
This should get moved down (with the other non-exported properties).  It also shouldn't have a json annotation (it's not exported, so I don't think it would be used anyway, but we definitely don't want this property sent to clients).